server/paymentProviders/privacy/index.ts

```diff
@@ -14,10 +14,10 @@ import VirtualCardModel from '../../models/VirtualCard';
 import { Transaction } from '../../types/privacy';
 import { CardProviderService } from '../types';
 
-const createExpense = async (
+const processTransaction = async (
   privacyTransaction: Transaction,
   opts?: { host?: any; collective?: any; hostCurrencyFxRate?: number },
-): Promise<any> => {
+): Promise<typeof models.Expense | undefined> => {
   const amount = privacyTransaction.settled_amount;
```

Comment:
What is our guarantee that `settled_amount` is in USD here. Is it really always the case? Even if the charge is in another currency?

Reply:
Yes, they don't even list currency on their schema.